app/services/teachers/induction_status.rb

```diff
@@ -39,6 +41,16 @@ def status_tag_kwargs
   def induction_status = status_tag_kwargs.fetch(:text)
   def induction_status_colour = status_tag_kwargs.fetch(:colour)
 
+  def completed?
+    %w[
+      Exempt
+      Passed
+      Failed
+      PassedInWales
```

Comment:
`PassedInWales` is a status in DQT but not in TRS, confirmed by @gunndabad.

We don't need to include it.